fdbserver/Status.actor.cpp

```diff
@@ -2481,20 +2481,6 @@ ACTOR static Future<JsonBuilderObject> blobGranulesStatusFetcher(
 			}
 		}
 		statusObj["number_of_key_ranges"] = totalRanges;
-
-		// Mutation log backup
-		state std::string mlogsUrl = wait(getMutationLogUrl());
```

Comment:
remove it fron status json. fdbbackup/fdbrestore can report those information more accurately.